src/test/java/edu/cornell/kfs/sys/util/CuXMLUnitTestUtils.java

```diff
@@ -0,0 +1,42 @@
+package edu.cornell.kfs.sys.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.custommonkey.xmlunit.DetailedDiff;
+import org.custommonkey.xmlunit.Diff;
+import org.custommonkey.xmlunit.Difference;
+import org.custommonkey.xmlunit.XMLUnit;
+import org.xml.sax.SAXException;
+
+public class CuXMLUnitTestUtils {
+    
+    private static final Logger LOG = LogManager.getLogger();
+    
+    public static void compareXML(File actualXmlFile, File expectedXmlFile) throws SAXException, IOException {
+        FileInputStream actualFileInputStream = new FileInputStream(actualXmlFile);
+        FileInputStream expectedFIleInputStream = new FileInputStream(expectedXmlFile);
```

Comment:
I'm not seeing any code that closes these streams. I would suggest using a try-with-resources block (or equivalent) to make sure the streams are properly closed afterwards.

Reply:
Also, for consistency with JUnit assertions, I would suggest changing the `compareXML()` args so that the expected value is the first arg and the actual value is the second arg.